src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -475,7 +478,12 @@ contract BancorVirtualSupplyBondingCurveFundingManager is
         (uint amountIssued, uint feeAmount) =
             _buyOrder(_receiver, _depositAmount, _minAmountOut);
         _addVirtualTokenAmount(amountIssued);
-        _addVirtualCollateralAmount(_depositAmount - feeAmount);
+        uint normalizedDepositAmount = _convertAmountToRequiredDecimal(
+            (_depositAmount - feeAmount),
+            collateralTokenDecimals,
```

Comment:
As the amounts now get normalized to 18 decimals, if we indeed use for example USDC with 6 decimals as collateral, and a UI would like to get the virtual supply, don't they get this now in 18 decimals. So the dAPP interfacing with the contract has to know to calculate the value back to it's orginal decimals like this right?

Maybe it makes more sense to always convert it back to the original decimals, i.e. in my example this would be 6. Wdyt @0xNuggan